### ISSUE TITLE
Accessibility Fixes Round 2

### ISSUE THREAD

**Issue by andrewmcgivery** [webcompat/webcompat.com#464]
**Homepage**
- [x] User Avatar missing alt text in header
- [x] Report Bug form radio buttons should be contained within a fieldset... will need CSS styling to more or less remove the default browser styles to keep the current look
- [x] If user has reported no issues, the "Nothing reported yet!" text is size 0 font

**About**
- [x] Missing H1... Should we make a h1 with .wc-sronly class applied or change "About" heading to H1 and then adjust rest of headings accordingly?

**Privacy**
- [x] Missing H1... Should we make a h1 with .wc-sronly class applied or change "Webcompat.com Privacy Policy" heading to H1 and then adjust rest of headings accordingly?
- [x] Jumps from H2 to H4... should change H4's to H3's and fix styles accordingly.

**Contributors**
- [x] Images should have more specific alt text other than "Browser Snapshot"
- [x] Jumps from H1 to H3's.... should change the H3's to H2's and style accordingly

Overall, we're in good shape. Just a few minor fixes and we'll be error free and warning free! (With the exception of a couple false positives)
